frontend/mina-signer/src/MinaSDK.ts

```diff
@@ -0,0 +1,356 @@
+"use strict";
+
+const JSOfOCaml_SDK = require("./client_sdk.bc.js");
+const minaSDK = JSOfOCaml_SDK.minaSDK;
+
+import {
+  network,
+  publicKey,
+  keypair,
+  privateKey,
+  signed,
+  payment,
+  stakeDelegation,
+  message,
+} from "./TSTypes";
+
+const defaultValidUntil = "4294967295";
+
+class Client {
+  private network: network;
+
+  constructor(options: { network: network }) {
```

Comment:
maybe allow uppercase, or mixed-case, e.g. `Mainnet`, by normalizing to lowercase?

Reply:
+1 .toLowerCase()